### PR TITLE
[IMP] mail: add message backlink on replyingto in composer

### PR DESCRIPTION
**Pull request by rapr-odoo**
**Current behavior before PR:**

There is no backlink to message on `Replying To` in header of Composer

**Desired behavior after PR is merged:**

There is a backlink to message

Task-2664839

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/components/composer/composer.scss [hunk 1]
```diff
@@ -75,7 +75,8 @@
     grid-area: core-header;
 }
 
-.o_Composer_cancelReply {
+.o_Composer_cancelReply,
+.o_Composer_replyingToMessage{
```
Comment (seb-odoo) [inline addons/mail/static/src/components/composer/composer.scss:79]: Avoid `,` in selectors this is a bit confusing and hard to maintain. Just duplicate this rule if necessary.

**Review by seb-odoo — COMMENTED** [review]
Hello, thank you for the PR!

Considering the time it took us to "fix" the issue, it's probably no longer worth it to do in v15, but just move the PR to master instead to be safe.

**Comment by rapr-odoo** [comment]
@seb-odoo Changes Done!
New PR Generated for master https://github.com/odoo/odoo/pull/94873

**Comment by seb-odoo** [comment]
Thank you, I'll check in master

**Pull request closed without merging**